### PR TITLE
Announce the mounting of engines

### PR DESCRIPTION
**Pull request by peterhurford**

### CODE REVIEW

**Comment by kirillseva** [comment]
Screenshot?

**Comment by kirillseva** [comment]
:+1:

**Comment by RyanNaughton** [comment]
LGTM

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: syberia
 Type: Package
 Title: Syberia
-Version: 0.6.1
+Version: 0.6.1.1
```
Comment (robertzk) [inline DESCRIPTION]: Notice that there is a [development version](https://github.com/syberia/syberia/releases). Can you work off that and tag 9001?

**Comment by peterhurford** [comment]
@kirillseva Screenshotted.
@robertzk Will do.

**Comment by robertzk** [comment]
Can you rebase the commit history to a single commit?

**Pull request closed without merging**